apps/mobile/src/screens/chat-screen/chat-screen.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from "react";
```

Comment:
```suggestion
import React from "react";
```
Import from ~/libs/hooks/hooks